pkg/utils/labels_annotations.go

```diff
@@ -75,6 +75,9 @@ const (
 	// or not, use for Secrets or ConfigMaps
 	WatchedLabelName = MetadataNamespace + "/reload"
 
+	// UserRoleLabeName the name of the label which tell if a Secret refers to a superuser database role or a default one
+	UserRoleLabelName = MetadataNamespace + "/user-role"
```

Comment:
Same comment as above, maybe "/user-class" or kind

Reply:
I thought about `userClass` too, but then I opted for `userType`. Had I read your input earlier, I would have named it `userClass`. :)

Reply:
UserType is nice and clear to understand !